src/components/diagrams/diagram-common.js

```diff
@@ -31,6 +31,10 @@ export const MAX_WIDTH_SUBSTATION = 1200;
 export const MAX_HEIGHT_SUBSTATION = 700;
 export const MAX_WIDTH_NETWORK_AREA_DIAGRAM = 1200;
 export const MAX_HEIGHT_NETWORK_AREA_DIAGRAM = 650;
+export const NETWORK_AREA_DIAGRAM_NB_MAX_VOLTAGE_LEVELS = 200;
+
+// To allow controls that are in the corners of the map to not be hidden in normal mode
+// (but they are still hidden in fullscreen mode)
```

Comment:
`// To allow controls that are in the corners of the map to not be hidden in normal mode`
`// (but they are still hidden in fullscreen mode)`

These comments should not be there anymore, they were removed in a recent merge.

Reply:
done